backend/src/main/kotlin/cz/loono/backend/schedule/ExaminationCancellationTask.kt

```diff
@@ -0,0 +1,33 @@
+package cz.loono.backend.schedule
+
+import cz.loono.backend.api.dto.ExaminationStatusDto
+import cz.loono.backend.api.service.ExaminationRecordService
+import cz.loono.backend.api.service.PreventionService
+import cz.loono.backend.db.repository.ExaminationRecordRepository
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+import java.time.LocalDateTime
+
+@Component("examCancel")
+class ExaminationCancellationTask @Autowired constructor(
```

Comment:
Make use of Spring's `Component` annotation? I.e.
```suggestion
class ExaminationCancellationTask(
```